docs/05_operations/02_commands/01_server-commands.md

```diff
@@ -895,21 +896,48 @@ SendIt -t FUND -f FUND.csv
 
 This reads the **FUND.csv** file in the local directory and inserts the data from the file into the FUND table.
 
-To modify records, you need to specify the key that will be used to identify the original record from each row in the csv file. If you want to modify a key field, you need to ensure the lookup key does not use this field; for example, you can't change an ID in the file and then modify on _BY_ID key.
+### Modifying and upserting
+
+:::warning
+Before making any changes to the database, you should switch off all processes.
```

Comment:
that might not be necessary in every case. It is valid to use SendIt whilst processes are up, and dataservers/consolidators will register updates as expected. Should this be a recommendation instead?

Reply:
I did think about going easier on this - best practice, or something like that. I'll change it. Thanks